docs/source/tutorial/inductive_lp.rst

```diff
@@ -0,0 +1,111 @@
+It is ensured that the relations sets of all inference graphs are subsets of their training graphs.
+
+
+Featurizing Unseen Entities
+-------------
+
+Training entity embeddings on the training graph is meaningless as those embeddings cannot be
+used at inference time.
+Instead, we need some universal featurizing mechanism which would build representations of both seen
+and unseen entities.
+In PyKEEN, there exist at least 2 such mechanisms depending on the availability of node descriptions.
+
+
+NodePiece
+~~~~~~~~~
+In the most basic case, unseen entities arrive without any features nor descriptions.
+We cater for this case using :class:`pykeen.nn.emb.NodePieceRepresentation` -
+since the set of relations at training and inference time is the same, NodePiece Representation
+will *tokenize* each entity through a subset of incident relation types.
+Out of computational reasons, NodePiece representations of `inductive_inference` entities
+(to be seen at inference time) can be pre-computed as well.
+The inductive version of NodePiece, :class:`pykeen.models.unimodel.InductiveNodePiece`, trains an encoder
+on top of the vocabulary of relational *tokens* that can be easily re-used at inference time.
+This way, we can obtain representations of unseen entities.
+`InductiveNodePiece` can be paired with any interaction function from PyKEEN where dimension of relation vectors
+is the same as dimension of final node vectors. Alternative interactions can be integrated with custom
+initialization of the relation representation module.
+
+Another example is NodePiece, which takes inspiration
+from tokenization we encounter in, e.g.. NLP, and represents each entity
+as a set of tokens. The implementation in PyKEEN,
+:class:`pykeen.nn.emb.NodePieceRepresentation`, implements a simple yet
+effective variant thereof, which uses a set of randomly chosen incident
+relations (including inverse relations) as tokens.
+
+.. seealso:: Include link to the Tutorials/Representations page
+
+
+Label-based Transformer Representation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If entity descriptions are available, the universal featurizing mechanism can
+be a language model accessible via :class:`pykeen.nn.emb.LabelBasedTransformerRepresentation`.
+At both training and inference time, fixed-size entity vectors are obtained after passing
+their textual descriptions through a pre-trained LM.
+This is work in progress and not yet available.
+TODO link to the Tutorial/Representations page
+
+
+Training & Evaluation
+---------------------
+Generally, training and evaluation of inductive models uses similar interfaces:
+sLCWA and LCWA training loops, and RankBasedEvaluator.
+The important addition of inductive interfaces is the `mode` argument. When set to `mode=train`,
+an inductive model has to invoke representations of the training graph, when set to `mode=valid`
+or `mode=test`, the model has to invoke representations of inference graphs.
```

Comment:
Should the modes be `transductive` and `inductive` instead? This would make the shared representations in `validation` and `testing` more explicit. (also note that most of the code-base uses `testing` instead of `test`)

Reply:
maybe the mode can also be an arbitrary string? (similar to a dictionary interface mapping keys to representations)

Reply:
I'd rather keep train/valid/testing modes since the next direct inductive use-case is when validation and test have their own disjoint inference graphs, and with explicit modes we support this use-case